File: Simulation of dropping dice/simulation.py
import matplotlib
import matplotlib.pyplot as plt
import random

x = [0, 100, 0]
y = [0, 0, 100]


plt.plot(x, y, ',')

curr_point = [random.randint(0,100), random.randint(0,100)]

for i in range(100000):
    j = random.randint(0,2)
    plt.plot(curr_point[0], curr_point[1], ',')
    curr_point[0] = (curr_point[0] + x[j])/2
    curr_point[1] = (curr_point[1] + y[j])/2

plt.show()

File: Simulation of dropping dice/simulation.m
x = rand(1,3) * 10;
y = rand(1,3) * 10;

curr_point = rand(1,2) * 10;


plot(x, y, 'o');


for i = 1:10000
    
    j = 1 + floor((rand(1) * 3));
    
    plot(curr_point(1), curr_point(2), 'o');
    
    curr_point(1) = (curr_point(1) + x(j))/2;
    curr_point(2) = (curr_point(2) + y(j))/2;
    
    
end


